app.json

```diff
@@ -0,0 +1,36 @@
+{
+  "name": "network",
+  "description": "The Mozilla Network site",
+  "addons": [
+    "heroku-postgresql:hobby-dev"
+  ],
+  "env": {
+    "HEROKU_APP_NAME": {
```

Comment:
https://devcenter.heroku.com/articles/github-integration-review-apps#heroku_app_name-and-heroku_parent_app_name

This, plus some changes I made to a few jsx components and the main.js file should let review apps automagically work